doc/source/whatsnew/v3.0.0.rst

```diff
@@ -593,6 +593,7 @@ Styler
 Other
 ^^^^^
 - Bug in :class:`DataFrame` when passing a ``dict`` with a NA scalar and ``columns`` that would always return ``np.nan`` (:issue:`57205`)
+- Bug in :func:`eval` including division ``\`` failed with a ``TypeError``. (:issue:`58748`)
```

Comment:
nit: division is `/`, not `\`. Can you also include it's with `` :class:`ExtensionArray` `` dtypes.

Reply:
Okay got it, I will look into this, Thanks.